modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -35,6 +35,51 @@ Because `/var` must be in place before a fresh installation of {op-system-first}
 If you follow the steps to create a separate `/var` partition in this procedure, it is not necessary to create the Kubernetes manifest and Ignition config files again as described later in this section.
 ====
 
+.Prerequisites
+
+* If container storage is on the root partition, then ensure that this root partition is mounted with the `pquota` option by including `rootflags=pquota` in the GRUB command line.
+
+* If the container storage is on a partition that is mounted by `/etc/fstab`, then include the following options in the file:
++
+[source,terminal]
+----
+/dev/sdb1      /var           xfs defaults,pquota 0 0
+----
+
+* If the container storage is on a partition that is mounted by `systemd`, then the following sample spec shows the mount option added:
```

Comment:
can omit "then" here - also not entirely clear to me if the user had to do something to make the spec look this way, or if it's just what they would see if the partition were mounted by systemd. And is it the "`var.mount` option"?

Reply:
@gpei Hey Gaoyun, can you answer this comment? I am unsure.

Reply:
It should be a MachineConfig that defines the systemd mount service, like the example file we mentioned in https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-user-infra.html#installation-disk-partitioning-upi-templates_installing-aws-user-infra 

```
3. Create a MachineConfig object and add it to a file in the openshift directory. For example, name the file 98-var-partition.yaml, change the disk device name to the name of the storage device on the worker systems, and set the storage size as appropriate. This attaches storage to a separate /var directory.

```